k8s/jiva/snapshot-cleanup.sh

```diff
@@ -0,0 +1,89 @@
+    #!/bin/bash
+
+    usage()
+    {
+        echo "Usage: bash snapshot-cleanup.sh <pv-name> <number_of_snapshots_to_delete>"
+        echo "Note: Before running this script kindly make sure there is no tmp.json file present in script execution directory"
+        exit 1
+    }
+
+    warning()
+    {
+        echo "WARNING: This script need to be run from client that have access to the cluster. This script can run for longer time based on the amount of data in the snapshots that need to be deleted. Please don't cancel the operation while it is running. If the script gets broken due to any network issue, you need to run following command [ bash snapshot-cleanup.sh <pv-name> restore_service ]"
```

Comment:
> this script can run for longer time based on the amount of data in the snapshots that need to be deleted.

The wording here is not quite correct.

Reply:
Also if the "progress" might belong its generally a good idea to give a sign of life every now and again like, by, for example, making use of a progress counter/bar or something to that extent.

Reply:
WARNING: Snapshot cleanup involves disconnecting the application from the storage. Also, while the snapshot cleanup is in progress - you will need to ensure that the connectivity to the Kubernetes Clusters is active. In case of unexpected disconnect, you will have to run the following command to restore the volume service. `snapshot-cleanup.sh <pv-name> restore_service`.